config/src/test/java/org/springframework/security/config/annotation/web/configurers/SessionManagementConfigurerServlet31Tests.java

```diff
@@ -100,15 +89,17 @@ public void changeSessionIdDefaultsInServlet31Plus() throws Exception {
 		CsrfToken token = repository.generateToken(request);
 		repository.saveToken(token, request, response);
 		request.setParameter(token.getParameterName(), token.getToken());
-		when(ReflectionUtils.findMethod(HttpServletRequest.class, "changeSessionId"))
-				.thenReturn(method);
 
 		loadConfig(SessionManagementDefaultSessionFixationServlet31Config.class);
 
 		springSecurityFilterChain.doFilter(request, response, chain);
 
-		verifyStatic(ReflectionUtils.class);
-		ReflectionUtils.invokeMethod(same(method), any(HttpServletRequest.class));
+
+		Assert.assertNotEquals(id, request.getSession().getId());
+		Assert.assertEquals(request.getParameter(token.getParameterName()), token.getToken());
```

Comment:
What are you wanting to verify by checking the request parameters? Since they aren't part of the session, they wouldn't be affected by session changes. 

Not sure if this is what you are going for, but to confirm that session fixation didn't squash anything the session, we'd need to add a session attribute, e.g. `request.getSession().setAttribute("custom", "value")`

Reply:
that was the idea, I changed that